src/plots/plots.js

```diff
@@ -786,6 +787,27 @@ function applyTransforms(fullTrace, fullData, layout) {
     var container = fullTrace.transforms,
         dataOut = [fullTrace];
 
+    var attributeSets = dataOut.map(function(trace) {
+
+        var arraySplitAttributes = [];
+
+        var stack = [];
+
+        function callback(attr, attrName, attrs, level) {
+
+            stack = stack.slice(0, level).concat([attrName]);
+
+            var splittableAttr = attr.valType === 'data_array' || attr.arrayOk === true;
+            if(splittableAttr) {
+                arraySplitAttributes.push(stack.slice());
+            }
+        }
+
+        crawler.crawl(trace._module.attributes, callback);
```

Comment:
Is the crawler able to catch _nested_ attributes like `error_y.array` declared [here](https://github.com/plotly/plotly.js/blob/b2b68cd8e8c401aaef3aefad10a52a4975ba6b58/src/traces/scatter/attributes.js#L357-L360) ?

If not, it's not a big deal. I'm not a big fan of the `_nestedAttribute` pattern anyway. We could simple `require` nested attribute in the root attribute modules to make this work.